runtime/compiler/env/J9PersistentInfo.cpp

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+// #if defined(J9VM_OPT_JITSERVER)
+#include "control/CompilationThread.hpp"
+#include "runtime/JITClientSession.hpp"
+// #endif
+#include "env/PersistentInfo.hpp"
```

Comment:
@mpirvu The includes in this file have some weird issues:
- If I uncomment `#if defined(J9VM_OPT_JITSERVER)`, then the code compiles but it compiles as if the environment variable is not set, i.e. none of the jitserver code in this file is compiled.
- If I move `PersistentInfo` include to the top of the file, then compilation fails with `‘getRemoteCompilationMode’ was not declared in this scope` message despite it being declared in `J9PersistentInfo.hpp`.

I don't know what might be causing this as I think I modified make/cmake files correctly and everything else compiles without problems. This problem persists when I build a clean OpenJ9 sdk so it's not a stale build issue.

Reply:
From my diagnostic, even for other files `J9VM_OPT_JITSERVER` is not defined. What defines it is `j9cfg.h`, so we need to include this header files in all files that use JITServer code.
In most of the places it works because we include a header which includes another header, etc, which includes j9cfg.h